diff --git a/cjs/misc.js b/cjs/misc.js
index d87e4532bedc01a58a8bb61fe1ea81f3025e5198..fcd5e19443760e72e052bdc336bd9513a32b676c 100644
--- a/cjs/misc.js
+++ b/cjs/misc.js
@@ -19,7 +19,15 @@ function uuid4() {
       return crypto.randomUUID().replace(/-/g, '');
     }
     if (crypto && crypto.getRandomValues) {
-      getRandomByte = () => crypto.getRandomValues(new Uint8Array(1))[0];
+      getRandomByte = () => {
+        // crypto.getRandomValues might return undefined instead of the typed array
+        // in old Chromium versions (e.g. 23.0.1235.0 (151422))
+        // However, `typedArray` is still filled in-place.
+        // @see https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#typedarray
+        const typedArray = new Uint8Array(1);
+        crypto.getRandomValues(typedArray);
+        return typedArray[0];
+      };
     }
   } catch (_) {
     // some runtimes can crash invoking crypto
diff --git a/esm/misc.js b/esm/misc.js
index 56856932b37f8d501743d5376184da1227a94cf5..c2f8e8590ddd3d4dbd3c44733de6e51f6fae4e1c 100644
--- a/esm/misc.js
+++ b/esm/misc.js
@@ -17,7 +17,15 @@ function uuid4() {
       return crypto.randomUUID().replace(/-/g, '');
     }
     if (crypto && crypto.getRandomValues) {
-      getRandomByte = () => crypto.getRandomValues(new Uint8Array(1))[0];
+      getRandomByte = () => {
+        // crypto.getRandomValues might return undefined instead of the typed array
+        // in old Chromium versions (e.g. 23.0.1235.0 (151422))
+        // However, `typedArray` is still filled in-place.
+        // @see https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues#typedarray
+        const typedArray = new Uint8Array(1);
+        crypto.getRandomValues(typedArray);
+        return typedArray[0];
+      };
     }
   } catch (_) {
     // some runtimes can crash invoking crypto